ui/components/ui/nickname-popover/nickname-popover.stories.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import Button from '../button';
+import NicknamePopover from '.';
+
+export default {
+  title: 'NicknamePopover',
+  id: __filename,
+};
+
+export const Default = () => {
+  const [showNicknamePopover, setShowNicknamePopover] = useState(false);
+
+  return (
+    <div>
+      <Button onClick={() => setShowNicknamePopover(true)}>
+        Open Nickname Popover
+      </Button>
+      {showNicknamePopover && (
+        <NicknamePopover
+          address="0x5e6DaAD1BE117e26590F9eEcD509336ABFBe5966"
```

Comment:
You can `import { text } from '@storybook/addon-knobs';` and then change this to `address={text('0x5e6DaAD1BE117e26590F9eEcD509336ABFBe5966')}` This will allow the text to be editable in the storybook component

Reply:
@danjm I have done `import { text } from '@storybook/addon-knobs';` and made a change to this:
` address={text('Address','0x5e6DaAD1BE117e26590F9eEcD509336ABFBe5966')}`
You can see changes here: b0b2b895cf388ec1988c6dc13718a450ff3a0f6a
Is it okey now?

Reply:
looks good